packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/StakeModal/StakeEthForm/index.tsx

```diff
@@ -0,0 +1,81 @@
+import styled from 'styled-components';
+import { Button } from '@trezor/components';
+import { Translation } from 'src/components/suite';
+import { useStakeEthFormContext } from 'src/hooks/wallet/useStakeEthForm';
+import { AvailableBalance } from '../AvailableBalance';
+import { FormFractionButtons } from 'src/components/suite/FormFractionButtons';
+import { Inputs } from './Inputs';
+import { Fees } from './Fees';
+import { isZero } from '@suite-common/wallet-utils';
+
+const Body = styled.div`
+    margin-bottom: 26px;
+`;
+
+const InputsWrapper = styled.div`
+    margin-top: 16px;
+    margin-bottom: 22px;
+`;
+
+const ButtonsWrapper = styled.div`
+    margin-top: 12px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+`;
+
+export const StakeEthForm = () => {
+    const {
+        account,
+        onSubmit,
+        handleSubmit,
+        formState: { errors, isSubmitting, isDirty },
+        isComposing,
+        setRatioAmount,
+        setMax,
+        watch,
+        clearForm,
+    } = useStakeEthFormContext();
+    const { formattedBalance, symbol } = account;
+    const hasValues = Boolean(watch('fiatInput') || watch('cryptoInput'));
+    // used instead of formState.isValid, which is sometimes returning false even if there are no errors
+    const formIsValid = Object.keys(errors).length === 0;
+    const areFractionButtonsDisabled = isZero(account.formattedBalance);
+
+    return (
+        <form onSubmit={handleSubmit(onSubmit)}>
+            <Body>
+                <AvailableBalance formattedBalance={formattedBalance} symbol={symbol} />
+
+                <ButtonsWrapper>
+                    <FormFractionButtons
+                        isDisabled={areFractionButtonsDisabled}
+                        setRatioAmount={setRatioAmount}
+                        setMax={setMax}
```

Comment:
Last thing:

In case I do not have 0.03 ETH and click on Set max, I get into negative balance

![Screenshot 2023-12-11 at 12 52 39](https://github.com/everstake/trezor-suite/assets/33235762/90389b02-dbff-40db-8ca8-e68949781e56)

Reply:
P.S. I've left intentionally. Thought it would be useful for a user to see exactly how much they need

Reply:
Okay, makes sense, lets keep it for now and I will raise a discussion about it